test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,63 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+            viewComponent.ViewData.Model = new object();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
```

Comment:
Isn't `model: null as object` the exact same as `model: null`? I mean, literally actually the same (in terms of resulting code)?